src/Pages/Platforms/platformTabs.tsx

```diff
@@ -31,9 +32,10 @@ export interface PlatformTabsProps extends RouteComponentProps {
  * Display tab bar and tab data for individual platforms
  * @param param0 React-Router props
  */
-export const PlatformTabs: React.SFC<PlatformTabsProps> = ({ match }) => {
+export const PlatformTabs: React.FC<PlatformTabsProps> = ({ match }) => {
   const { id } = match.params
   const { path } = match
+  const unit_system = useUnitSystem()
```

Comment:
Codacy found an issue: [Identifier 'unit_system' is not in camel case.](https://app.codacy.com/manual/gmri/Neracoos-1-Buoy-App/pullRequest?prid=6166252)